pkg/storageos/deploy.go

```diff
@@ -1087,6 +1089,14 @@ func (s *Deployment) addNodeAffinity(podSpec *v1.PodSpec) {
 	}
 }
 
+// addToleration adds toleration to the given pod spec from the cluter
+// spec Toleration.
```

Comment:
Keyword typos.
`Toleration` -> `Tolerations`.
It's incorrect in `addNodeAffinity` comment as well. Should be `NodeSelectorTerms`, not `NodeSelectorLabel`.